langwatch/src/optimization_studio/components/Publish.tsx

```
@@ -96,6 +107,44 @@ export function Publish({ isDisabled }: { isDisabled: boolean }) {
   );
 }
 
+const exportWorkflow = async (
+  publishedWorkflow: Workflow,
+  datasetData?: Dataset & { datasetRecords: DatasetRecord[] }
+) => {
+  let dsl = publishedWorkflow;
+
+  if (
+    datasetData &&
+    datasetData.datasetRecords &&
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)